api/client/clientModel.js

```diff
@@ -13,7 +13,11 @@ const findAll = () => {
 };
 
 const findById = async (id) => {
-  return db('clients').where({ id }).first();
+  const client = db('clients').where({ id }).first();
+
+  client.gender = db('genders').where(client.gender_id, 'genders.id').first();
```

Comment:
same concern here, what if the promise doesn't resolve before line 20 returns client?